src/server/services/wms/qgswmsgetprint.cpp

```diff
@@ -25,51 +25,36 @@
 namespace QgsWms
 {
   void writeGetPrint( QgsServerInterface *serverIface, const QgsProject *project,
-                      const QString &version, const QgsServerRequest &request,
+                      const QString &, const QgsServerRequest &request,
                       QgsServerResponse &response )
   {
-    QgsServerRequest::Parameters params = request.parameters();
-
-    Q_UNUSED( version );
-
-    QgsWmsParameters wmsParameters( QUrlQuery( request.url() ) );
+    const QgsWmsParameters wmsParameters( QUrlQuery( request.url() ) );
     QgsRenderer renderer( serverIface, project, wmsParameters );
 
-    QString format = params.value( "FORMAT" );
+    const QgsWmsParameters::Format format = wmsParameters.format();
```

Comment:
Good news -- it looks like soon we'll be able to run a clang-tidy fixit over the whole code and automatically add all these missing consts: https://reviews.llvm.org/D45444

When we've done that let's crack down on this and make it standard QGIS coding practice to constify anything which can be.

Reply:
> When we've done that let's crack down on this and make it standard QGIS coding practice to constify anything which can be.

Yesss!!!!!